docs/examples/basic_json__CompatibleType.output

```diff
@@ -2,7 +2,7 @@
 {"one":1,"three":3,"two":2}
 {"one":1.2,"three":3.4,"two":2.3}
 {"one":true,"three":false,"two":true}
-{"one":true,"three":true,"two":true}
+{"one":true,"three":false,"two":true}
```

Comment:
Is it expected that 3.11.0 changed which of the two `"three"` elements were included, and 3.11.2 changed it back?

Reply:
Depends on the STL. :man_shrugging:

Reply:
Just wondering if the change is an indication of a change in the library, or a change in a CI setting, or if this was a manual change, or if we can expect it to flip-flop in the future.

Reply:
This was generated with libstdc++ (going by `meta.output`) and not libc++.

Reply:
Expect it to flip-flop based on what toolchain Niels uses.

Reply:
So this file is generated by the release process, and it's reasonable for it to change from time to time during that process.

Reply:
Sorry about this inconvenience. I had to adjust the release script several times, and it  is still a bit shaky. The examples did not work out of the box so I had to set GCC as compiler. Then there was an update between the releases. Even worse, the behavior differs when I execute the script in my IDE vs. in my terminal (because the paths differs). I hope to get this fixed soon, but for the time being, the behavior will likely stay this way.

Reply:
As long as it's explainable, that's fine by me.